apps/asn1pars.c

```diff
@@ -226,8 +226,8 @@ int asn1parse_main(int argc, char **argv)
             ASN1_TYPE *atmp;
             int typ;
             j = atoi(sk_OPENSSL_STRING_value(osk, i));
```

Comment:
I really think this should be converted to using `strtoul`.

`strtoul` is part of C89, which is our minimum requirement (because it's ANSI C), so using it isn't an issue.

Reply:
(and before you say it, yes there's a gazillion other places where we use `atoi`...  let's make this one less of them)

Reply:
Okay, if you like that better.
I think I should use strtol(..,NULL,0) because that is used in opt_long.

Reply:
now gazillion-3 left :)

Reply:
Last time I'll waste time repeating:  apps should use opt_int or one of its ilk that are in apps/opt.c